src/main/java/dev/ebrydeu/spring_boot_library/exception/CustomExceptions.java

```diff
@@ -0,0 +1,57 @@
+package dev.ebrydeu.spring_boot_library.exception;
+
+public class CustomExceptions {
+    public static class UserNotFoundException extends RuntimeException {
+        public UserNotFoundException(String message) {
+            super(message);
+        }
+    }
```

Comment:
Maybe make this more generic ? Like `NotFoundException`. The reason that if we have more than one Entity then each need own `XxxNotFoundException`.

Reply:
great I will fix this, thanks! :D